src/services/game.service.ts

```diff
@@ -37,7 +40,53 @@ class GameService {
 
   private async wordWasUsedInGame(word: string, gameId: string): Promise<boolean> {
     const rounds = await roundService.getAllByGameId(gameId);
-    return rounds.some(round => round.words.includes(word));
+    return rounds.some(round => round.words.map(obj => obj.word).includes(word));
+  }
+
+  async start(id: string) {
+    await gameRepository.start(id);
+    const game = await this.getById(id);
+    const roundId = await this.createRound(game);
+    game.rounds.push(roundId);
+    await this.update(game);
+    //TODO provide chat
+  }
+
+  private async createRound(game: IGame): Promise<string> {
+    const roundId = await roundService.create({
+      teamId: game.currentTeam,
+      hostId: await this.getHostIdFromTeam(game.currentTeam, game._id!),
+      chatId: await chatService.create(),
+      finishedAt: this.getFinishDate(new Date(), game.options.roundTime),
+      currentWord: await this.getRandomWord(game._id!)
```

Comment:
I am not sure, the infinite loop will  or will not happen.

Reply:
I said about this on last meeting, but for now it will happen.. So I will update this branch to fix it